### DISCUSSION TITLE
是否可以提供选项，以关闭“在检测到解除番剧区域限制的脚本在运行时, 本脚本将停止运行以避免兼容性问题”功能？

### DISCUSSION THREAD

**Discussion by abbaccadd0** [the1812/Bilibili-Evolved#4100] [功能建议 / Ideas]
🤔如题，是否可以提供一个开关关闭“在检测到解除番剧区域限制的脚本在运行时, 本脚本将停止运行以避免兼容性问题”的功能
我查看了[#2704](https://github.com/the1812/Bilibili-Evolved/issues/2704)的相关问题，我在旧版本同时使用这两个脚本时也遇到了显示的问题
——但是其实不影响播放
现在的问题成了：我需要关闭区域限制，然后在我打开需要解锁区域的视频之前手动打开，看完之后再手动关闭，否则所有视频页面均无法加载Bilibili-Evolved
这甚至比[#2704](https://github.com/the1812/Bilibili-Evolved/issues/2704)更加恼人，每次看其他地区的番剧都需要至少额外点四下才能正常使用两个脚本
所以有没有可能为这个关闭机制添加一个开关呢？我可以忍受显示错位之类的，只要视频可以正常播放，但是来回开关脚本实在是麻烦了些。

**Comment by the1812** [comment]
在搞了

**Reply by the1812** [reply]
https://github.com/the1812/Bilibili-Evolved/discussions/4095#discussioncomment-5432331

**Reply by tkp206093** [reply]
> 在搞了

我更特別 看普通視頻都直接被禁用了 不知是否判定有問題？
